examples/bri-3/src/bri/zkp/components/merkleTree/merkleTree.ts

```diff
@@ -0,0 +1,74 @@
+import { Tree } from "./tree"
+import { IMerkleTree} from "./merkleTree.interface";
+import { PreciseProofs } from "ew-precise-proofs-js";
+
+export class MerkleTree implements IMerkleTree {
+  tree: Tree;
+
+  constructor(document: any){
+    const merkleTree: Tree = this.createTree(document);
+
+    this.tree.leaves = merkleTree.leaves;
+    this.tree.root = merkleTree.root; 
+  }
+
+  private createTree(document: any): Tree {
+    try {
+      var tree: Tree;
```

Comment:
please utilize `let`

Reply:
i think you can just init class variable tree inside createTree method (maybe call it initTree), so you don't need separate tree variable here, but just a preference
if you leave it like this, you dont need to declare tree here you can just return it and declare there